doc/shared/Install.rst

```diff
@@ -1016,9 +1016,13 @@ illustration only.
 
 .. cmakeoption:: USE_GENERIC_MATH
 
-   Use generic (``stdc``) math libraries
+   Use generic double precision math functions.  This is recommend when
```

Comment:
Change back to "Use generic (``stdc``) math libraries" . The rest that was added is great and explains what happens in C90. Add one more line at the end that notes that when it is ``OFF`` then `libm` wont be linked to.

Reply:
@balos1 currently `libm` is always linked to on UNIX or UNIX-like systems but not on others